source/_integrations/matter.markdown

```diff
@@ -23,182 +23,241 @@ ha_platforms:
+One of the great things about Matter is that you can have both Wi-Fi and Thread based devices on the same controller.
+Next to actual devices (like actors or sensors), you will also see bridges. The bridge connects the network over Ethernet or Wi-Fi and bridges multiple devices into a Matter network. A great example is the Philips Hue V2 bridge, which is a Zigbee hub and a Matter bridge. This bridge exposes all Zigbee devices already connected to the bridge as Matter devices on the network. Also, Aqara, SwitchBot, and IKEA have launched such Hub devices.
 
-<p class='note'>
-The Matter protocol relies on (local) IPv6 and mDNS (multicast traffic) which should be able to travel freely in your network. Matter devices (and any Thread Border routers) must be on the same LAN/VLAN as Home Assistant. Implementations like mDNS reflectors usually do more harm than good.
-</p>
+<div class='note'>
+Home Assistant, as a Matter controller, only supports **control** of Matter devices. Home Assistant is not a bridge itself and it cannot turn existing devices within Home Assistant into Matter compatible devices.
+</div>
+
+## Thread
+
+Matter goes hand-in-hand with (but is not the same as) [Thread](<https://en.wikipedia.org/wiki/Thread_(network_protocol)>), which is a low power Radio mesh networking techology. Much like Zigbee, but with the key difference that it is _IP-addressable_, making it the perfect companion transport for Matter.
+
+Thread devices become directly addressable by Matter controllers (such as Home Assistant) thanks to the use of so-called Thread Border Routers, which are in fact just devices that are both within your network and have a Thread chip builtin and thus act as a "router" between the Thread radio signal and your local network. These border routers (you will probably end up having multiple of them in your house) make sure that your Thread-based devices are reachable on your regular network and thus can be controlled with Matter. Examples of Thread Borders routers are the Apple TV 4K, HomePod (gen 2 or Mini), and the Google Nest Hub V2, so devices that you may already own. Besides that, all kind of other border routers are available, built-in to hardware appliances or software solutions based on OpenThread Border Router, such as the add-on we provide to use with the built-in Zigbee/Thread chip of the [Home Assistant Yellow](/yellow/) or the [Home Assistant SkyConnect](/skyconnect/) dongle.
+
+To use any Thread-based devices on a Matter controller, you need to have at least one Thread Border router device within range of the device.
+More info about Thread and diagnosing Thread networks and Border routers, see the [Thread](/integrations/thread/) integration.
```

Comment:
Some of these things duplicate what is written in the Thread integration, or should be part of the Thread integration. I'd go shorter here, and link to the Thread integration instead.

```suggestion
Matter goes hand-in-hand with (but is not the same as) {% term Thread %}. Thread is is a low power Radio mesh networking technology much like Zigbee. Unlike Zigbee, Thread is only a communication protocol that is _IP-addressable_. This makes it the perfect companion transport for Matter.

To use Thread based Matter devices you'll need a Thread border router such as Apple TV 4K, HomePod (gen 2 or Mini), the Google Nest Hub (2nd Gen) or Home Assistant add-ons based on the OpenThread Border Router.

Read more about Thread in the [Thread integration documentation](/integrations/thread/).
```

Reply:
Yes, that is on purpose because there's a lot of mixup/confusion hence the somewhat duplicated text. People use Thread as transport for Matter, they;re not installing Thread itself, hence I think the info should be here (for now).

Maybe we can remove it later when the dust settles. Can we perhaps copy some of the text to the Thread integration ?

Reply:
I am not against a bit of redundancy in the documentation, as a user it is always burdensome to visit several documentation pages to do one thing.
Should we replace all the occurrences of the word `Thread` by `{% term Thread %}`  actually ? (e.g. line 51, 53, 55 and 56)

Reply:
Agree, a duplicate is fine in this case. The subject is just to difficult at this point for people to grasp.

@Vinalti The use of the term plugin should be used sparingly. We don't want to link every word or occurrence.

Reply:
going to leave it as-is for now.